src/components/timeline/TrackContextMenu.js

```diff
@@ -188,10 +195,18 @@ class TimelineTrackContextMenu extends PureComponent<Props> {
         data={{ trackIndex }}
         onClick={this._toggleGlobalTrackVisibility}
         attributes={{
-          className: classNames({ checkable: true, checked: !isHidden }),
+          className: classNames('timelineTrackContextMenuItem', {
+            checkable: true,
+            checked: !isHidden,
+          }),
+          title,
         }}
       >
-        {globalTrackNames[trackIndex]}
+        <span>{globalTrackNames[trackIndex]}</span>
+        <span className="timelineTrackContextMenuSpacer" />
```

Comment:
I wonder if we can do the same output without adding additional span here.
How about removing this and adding these properties to `timelineTrackContextMenuPid`:
```
flex: 1;
margin-left: 20px;
text-align: right;
```
I would do something like it because I don't like additional elements that much but not sure if it's better or not, your call :)

Reply:
I started out trying to solve this without the extra spacer. But I always got problems whenever the scrollbar on the right appeared. Here's how it looks with the scrollbar when I try, for instance, your suggestion:

![Screenshot-2019-8-12 Firefox 67 0 - Windows NT 10 0; Win64; x64 - 2 8 2019, 3 14 08 PM UTC (public) - Firefox profiler](https://user-images.githubusercontent.com/1911402/62855855-9a1a4b00-bcf3-11e9-9362-11f6c226710b.png)

With the spacer I always get enough space to draw the text, and it shrinks slightly when the scrollbar appears so that it still looks fine.
I kept the spacer with an explanatory comment. But CSS is not my strength, so please let me know if I can solve this better :-)

Reply:
Ah, I didn't test with the scrollbar before. Okay then, let's keep it that way.